app/presenters/tree_builder_policy_simulation.rb

```diff
@@ -81,9 +80,8 @@ def get_active_caption(node)
   def policy_nodes(parent)
     parent[:policies].reject { |node| skip_node?(node) }.sort_by { |a| a["description"] }.map do |node|
       active_caption = get_active_caption(node)
-      name = "<b>" + _("Policy%{caption}: ") % {:caption => active_caption} + "</b> #{node['description']}"
       {:id         => node['id'],
-       :text        => name.html_safe,
+       :text        => prefixed_title(_('Policy%{caption}') % {:caption => active_caption}, node['description']),
```

Comment:
@skateman If you can, please re-model the logic here so that `get_active_caption()`  returns
the concatenated string, that we don't have to do `_('Policy%{caption}') % {:caption => active_caption}`

Either here or in a separate PR. Thanks.